src/main/java/frc/robot/subsystems/climber/Climber.java

```diff
@@ -0,0 +1,54 @@
+package frc.robot.subsystems.climber;
+
+import com.ctre.phoenix.motorcontrol.NeutralMode;
+import com.ctre.phoenix.motorcontrol.can.WPI_VictorSPX;
+import com.spikes2212.command.genericsubsystem.MotoredGenericSubsystem;
+import com.spikes2212.command.genericsubsystem.commands.MoveGenericSubsystem;
+import edu.wpi.first.wpilibj.shuffleboard.Shuffleboard;
+
+import java.util.function.Supplier;
+
+public class Climber extends MotoredGenericSubsystem {
+
+    public static final double MAX_SPEED = 0.5;
+    public static final double MIN_SPEED = -0.5;
+
+    /**
+     * The speed in which the arms will move up to catch the bar (in order to climb).
+     *
+     * <p>A game that this example subsystem can be used in is 2020 FRC INFINITE RECHARGE</p>
+     */
+    public final Supplier<Double> UP_SPEED = rootNamespace.addConstantDouble("up speed", 0.3);
+
+    /**
+     * The speed in which the arms will move down in order to take the robot up after they caught the bar.
+     */
+    public final Supplier<Double> DOWN_SPEED = rootNamespace.addConstantDouble("down speed", -0.4);
```

Comment:
you don't need `MAX_SPEED` and `MIN_SPEED` if you have `UP_SPEED` and `DOWN_SPEED`